### ISSUE TITLE
setting `stars = FALSE` doesn't remove stars in the table

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/correlation#136]
@strengejacke Do you think this is coming from `insight`?

``` r
library(correlation)

display(correlation(subset(mtcars, select = c("wt", "mpg")),
                    format = "html", stars = FALSE))
```

<table>
<caption>Correlation Matrix (pearson-method)</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Parameter1</th>
<th style="text-align: center;">Parameter2</th>
<th style="text-align: center;">r</th>
<th style="text-align: center;">95% CI</th>
<th style="text-align: center;">t(30)</th>
<th style="text-align: center;">p</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">wt</td>
<td style="text-align: center;">mpg</td>
<td style="text-align: center;">-0.87</td>
<td style="text-align: center;">(-0.93, -0.74)</td>
<td style="text-align: center;">-9.56</td>
<td style="text-align: center;">&lt; .001***</td>
</tr>
</tbody>
</table>

Correlation Matrix (pearson-method)

p-value adjustment method: Holm (1979)
Observations: 32

<sup>Created on 2021-02-07 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

**Comment by DominiqueMakowski** [comment]
It might be somewhere here https://github.com/easystats/correlation/blob/master/R/print.easycormatrix.R

Does display call print? Or is it a generic? In which case we can make it run format()?

**Comment by strengejacke** [comment]
display() calls either print_html() or print_md()

**Comment by strengejacke** [comment]
Not sure, but I think it's not an insight-thing. Looks like "stars" is not passed down to further format-functions...
https://github.com/easystats/correlation/blob/be50dba47cd0b5a2ed8b10a452ecb73a61dd8049/R/print.easycormatrix.R

**Comment by DominiqueMakowski** [comment]
Should be fixed for matrices:

``` r
x <- correlation::correlation(iris)

summary(x, stars=FALSE)
#> Parameter    | Petal.Width | Petal.Length | Sepal.Width
#> -------------------------------------------------------
#> Sepal.Length |        0.82 |         0.87 |       -0.12
#> Sepal.Width  |       -0.37 |        -0.43 |            
#> Petal.Length |        0.96 |              |            
#> 
#> p-value adjustment method: Holm (1979)

x <- summary(x)
x
#> Parameter    | Petal.Width | Petal.Length | Sepal.Width
#> -------------------------------------------------------
#> Sepal.Length |     0.82*** |      0.87*** |       -0.12
#> Sepal.Width  |    -0.37*** |     -0.43*** |            
#> Petal.Length |     0.96*** |              |            
#> 
#> p-value adjustment method: Holm (1979)

format(x, stars=FALSE)
#>      Parameter Petal.Width Petal.Length Sepal.Width
#> 1 Sepal.Length        0.82         0.87       -0.12
#> 2  Sepal.Width       -0.37        -0.43            
#> 3 Petal.Length        0.96
```

<sup>Created on 2021-02-10 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by DominiqueMakowski** [comment]
``` r
x <- correlation::correlation(iris)
x
#> # Correlation table (pearson-method)
#> 
#> Parameter1   |   Parameter2 |     r |         95% CI | t(148) |         p
#> -------------------------------------------------------------------------
#> Sepal.Length |  Sepal.Width | -0.12 | [-0.27,  0.04] |  -1.44 | 0.152    
#> Sepal.Length | Petal.Length |  0.87 | [ 0.83,  0.91] |  21.65 | < .001***
#> Sepal.Length |  Petal.Width |  0.82 | [ 0.76,  0.86] |  17.30 | < .001***
#> Sepal.Width  | Petal.Length | -0.43 | [-0.55, -0.29] |  -5.77 | < .001***
#> Sepal.Width  |  Petal.Width | -0.37 | [-0.50, -0.22] |  -4.79 | < .001***
#> Petal.Length |  Petal.Width |  0.96 | [ 0.95,  0.97] |  43.39 | < .001***
#> 
#> p-value adjustment method: Holm (1979)
#> Observations: 150
print(x, stars=FALSE)
#> # Correlation table (pearson-method)
#> 
#> Parameter1   |   Parameter2 |     r |         95% CI | t(148) |      p
#> ----------------------------------------------------------------------
#> Sepal.Length |  Sepal.Width | -0.12 | [-0.27,  0.04] |  -1.44 | 0.152 
#> Sepal.Length | Petal.Length |  0.87 | [ 0.83,  0.91] |  21.65 | < .001
#> Sepal.Length |  Petal.Width |  0.82 | [ 0.76,  0.86] |  17.30 | < .001
#> Sepal.Width  | Petal.Length | -0.43 | [-0.55, -0.29] |  -5.77 | < .001
#> Sepal.Width  |  Petal.Width | -0.37 | [-0.50, -0.22] |  -4.79 | < .001
#> Petal.Length |  Petal.Width |  0.96 | [ 0.95,  0.97] |  43.39 | < .001
#> 
#> p-value adjustment method: Holm (1979)
#> Observations: 150
format(x, stars=FALSE)
#>     Parameter1   Parameter2     r         95% CI t(148)      p
#> 1 Sepal.Length  Sepal.Width -0.12 [-0.27,  0.04]  -1.44 0.152 
#> 2 Sepal.Length Petal.Length  0.87 [ 0.83,  0.91]  21.65 < .001
#> 3 Sepal.Length  Petal.Width  0.82 [ 0.76,  0.86]  17.30 < .001
#> 4  Sepal.Width Petal.Length -0.43 [-0.55, -0.29]  -5.77 < .001
#> 5  Sepal.Width  Petal.Width -0.37 [-0.50, -0.22]  -4.79 < .001
#> 6 Petal.Length  Petal.Width  0.96 [ 0.95,  0.97]  43.39 < .001
#>                Method n_Obs
#> 1 Pearson correlation   150
#> 2 Pearson correlation   150
#> 3 Pearson correlation   150
#> 4 Pearson correlation   150
#> 5 Pearson correlation   150
#> 6 Pearson correlation   150

correlation::correlation(iris, stars=FALSE)  # This doesn't work... but should it?
#> # Correlation table (pearson-method)
#> 
#> Parameter1   |   Parameter2 |     r |         95% CI | t(148) |         p
#> -------------------------------------------------------------------------
#> Sepal.Length |  Sepal.Width | -0.12 | [-0.27,  0.04] |  -1.44 | 0.152    
#> Sepal.Length | Petal.Length |  0.87 | [ 0.83,  0.91] |  21.65 | < .001***
#> Sepal.Length |  Petal.Width |  0.82 | [ 0.76,  0.86] |  17.30 | < .001***
#> Sepal.Width  | Petal.Length | -0.43 | [-0.55, -0.29] |  -5.77 | < .001***
#> Sepal.Width  |  Petal.Width | -0.37 | [-0.50, -0.22] |  -4.79 | < .001***
#> Petal.Length |  Petal.Width |  0.96 | [ 0.95,  0.97] |  43.39 | < .001***
#> 
#> p-value adjustment method: Holm (1979)
#> Observations: 150
```

<sup>Created on 2021-02-10 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by IndrajeetPatil** [comment]
Thanks! I think that was the last fixing needed before cran submission :)

**Comment by DominiqueMakowski** [comment]
though checks are not passing?

**Comment by IndrajeetPatil** [comment]
Let me check.

**Comment by IndrajeetPatil** [comment]
Okay, everything looks good now.

Some github actions fail because `easystats` packages are not available for 3.5 and 3.4, but that's to be expected.
